index.js

```diff
@@ -52,7 +53,11 @@ module.exports = function NuxtTypeScript(moduleOptions) {
     loader: "babel-loader",
     options: Object.assign(
       {
-        plugins: ["transform-vue-jsx"]
+        presets: [
+          [
+            require.resolve('babel-preset-vue-app')
```

Comment:
1. `babel-preset-vue-app` needs to be added to the project dependencies. Please use `yarn add babel-preset-vue-app`
2. You shouldn't need to use `require.resolve` here—just a simple string with the "babel-preset-" omitted ie `presets: ["vue-app"]`
3. Since `babel-plugin-transform-vue-jsx` is no longer needed, please remove it from the project dependencies. `yarn remove babel-plugin-transform-vue-jsx`